man/rpm-ostree.xml

```diff
@@ -692,6 +692,20 @@ Boston, MA 02111-1307, USA.
           </para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><command>countme</command></term>
+
+        <listitem>
+          <para>
+            This commands implements DNF Count Me support in rpm-ostree. This
+            is a hidden command that should only be called from the
```

Comment:
I think we should actually document the systemd unit and not the entrypoint.  Which would argue for `man/rpm-ostree-countme.service.xml`?  And then `Documentation=man:rpm-ostree-countme.service` in the systemd unit.

(Though really the content of that man page is basically identical to `docs/countme.md`...so perhaps we could figure out how to ship that as a man page?)